### ISSUE TITLE
Error when running ng build

### ISSUE THREAD

**Issue by gschurck** [primefaces/sakai-ng#7]
I encountered this error when running `ng build`, just after cloning. `ng serve` is working fine
 
```
./node_modules/css-loader/dist/runtime/api.js - Error: Module build failed (from ./node_modules/@ngtools/webpack/src/ivy/index.js):
Transform failed with 1 error:
error: Invalid version: "15.2-15.3"

./node_modules/css-loader/dist/runtime/noSourceMaps.js - Error: Module build failed (from ./node_modules/@ngtools/webpack/src/ivy/index.js):
Transform failed with 1 error:
error: Invalid version: "15.2-15.3"

./src/main.ts - Error: Module build failed (from ./node_modules/@ngtools/webpack/src/ivy/index.js):
Transform failed with 1 error:
error: Invalid version: "15.2-15.3"

./src/polyfills.ts - Error: Module build failed (from ./node_modules/@ngtools/webpack/src/ivy/index.js):
Transform failed with 1 error:
error: Invalid version: "15.2-15.3"

./src/styles.scss - Error: Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
HookWebpackError: Module build failed (from ./node_modules/@ngtools/webpack/src/ivy/index.js):
Transform failed with 1 error:
error: Invalid version: "15.2-15.3"
    at tryRunOrWebpackError (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\HookWebpackError.js:88:9)
    at __webpack_require_module__ (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\Compilation.js:4979:12)
    at __webpack_require__ (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\Compilation.js:4936:18)
    at Module.<anonymous> (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\css-loader\dist\cjs.js??ruleSet[1].rules[3].rules[0].oneOf[1].use[1]!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\postcss-loader\dist\cjs.js??ruleSet[1].rules[3].rules[0].oneOf[1].use[2]!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\resolve-url-loader\index.js??ruleSet[1].rules[3].rules[1].use[0]!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\sass-loader\dist\cjs.js??ruleSet[1].rules[3].rules[1].use[1]!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\src\styles.scss:5:111)
    at C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\javascript\JavascriptModulesPlugin.js:432:11
    at Hook.eval [as call] (eval at create (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:7:1)
    at C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\Compilation.js:4981:39
    at tryRunOrWebpackError (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\HookWebpackError.js:83:7)
    at __webpack_require_module__ (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\Compilation.js:4979:12)
    at __webpack_require__ (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\Compilation.js:4936:18)
-- inner error --
Error: Module build failed (from ./node_modules/@ngtools/webpack/src/ivy/index.js):
Transform failed with 1 error:
error: Invalid version: "15.2-15.3"
    at Object.<anonymous> (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\@angular-devkit\build-angular\src\babel\webpack-loader.js??ruleSet[1].rules[1].use[0]!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\@ngtools\webpack\src\ivy\index.js!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\css-loader\dist\runtime\noSourceMaps.js:1:7)
    at C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\javascript\JavascriptModulesPlugin.js:432:11
    at Hook.eval [as call] (eval at create (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:7:1)
    at C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\Compilation.js:4981:39
    at tryRunOrWebpackError (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\HookWebpackError.js:83:7)
    at __webpack_require_module__ (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\Compilation.js:4979:12)
    at __webpack_require__ (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\Compilation.js:4936:18)
    at Module.<anonymous> (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\css-loader\dist\cjs.js??ruleSet[1].rules[3].rules[0].oneOf[1].use[1]!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\postcss-loader\dist\cjs.js??ruleSet[1].rules[3].rules[0].oneOf[1].use[2]!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\resolve-url-loader\index.js??ruleSet[1].rules[3].rules[1].use[0]!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\sass-loader\dist\cjs.js??ruleSet[1].rules[3].rules[1].use[1]!C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\src\styles.scss:5:111)
    at C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\webpack\lib\javascript\JavascriptModulesPlugin.js:432:11
    at Hook.eval [as call] (eval at create (C:\Users\guillaume.schurck\Desktop\TOTEM\sakai-ng\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:7:1)
```

**Comment by gschurck** [comment]
Okay it was happening because I chose to use the same Angular CLI version from the project, 13.0.4.
But it's fixed in the last versions, works with 13.3.0.

**Comment by cetincakiroglu** [comment]
Hi,

This problem is caused by Safari versions. To overcome that,

`
not ios_saf 15.2-15.3
not safari 15.2-15.3
`
paste this into the .browserlistsrc file
